<h3>Order item has failed</h3>
<p>Task: {{ name }} (id: {{ id }} }})</p>
<p>The following details are available:
    <ul>

    </ul>
</p>
<p>Exception: {{ exc }}</p>
<p>Exception: {{ highlighted_exc }}</p>
<p>Calling args: {{ args }}</p>
<p>Calling kwargs: {{ kwargs }}</p>
<p>Full traceback: {{ traceback }}</p>
